index.html

```diff
@@ -760,7 +760,7 @@ <h2>
         </dt>
         <dd>
           <p>
-            Language or behaviour that may appear kind or helpful but conveys a feeling of superiority or condescension.
+            Language or behaviour that may appear kind or helpful but conveys a feeling of superiority or condescension. <cite><a href="https://languages.oup.com/">Adapted from the Oxford Languages Dictionary.</a></cite>
```

Comment:
It's a shame that access to the dictionary appears to be behind a paywall - the cited link is not to a dictionary but to a landing page for Oxford Languages. Not sure what change could improve that other than using a definition from a more open dictionary.

Reply:
You can access the definition by googling "patronizing", which pops up the google dictionary feature powered by Oxford. There is a direct URL but it's a bit messy, I could use it though.

Reply:
I guess a Google link may at some point in the future not resolve to Oxford Languages, since they could switch provider at any time. Hmm, tricky, I don't have a good alternative suggestion.